app/views/guidance/entry.scala.html

```diff
@@ -0,0 +1,95 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.helpers.Title
+@import views.html.components.gds._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import config.AppConfig
+
+@this(
+    govukLayout: gdsMainTemplate,
+    pageTitle: pageTitle,
+    heading: heading,
+    paragraphBody: paragraphBody,
+    link: link,
+    govukButton: GovukButton,
+    bulletList: bulletList,
+    numberedList: numberedList,
+    sectionHeader: sectionHeader,
+    appConfig: AppConfig
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@guidance = @{ appConfig.guidance }
+
+@govukLayout(title = Title("guidance.entry.title")) {
+
+    @pageTitle(messages("guidance.entry.title"))
+
+
+@* Section 1 *@
+
+    @paragraphBody(messages("guidance.entry.section.1.paragraph.1"))
+
+    @paragraphBody(messages("guidance.entry.section.1.paragraph.2", link(
+        text = messages("guidance.entry.section.1.paragraph.2.link.1"),
+        call = Call("GET", guidance.appointSomeoneToDealWithCustomsOnYourBehalf)
+    )))
+
+    @paragraphBody(messages("guidance.entry.section.1.paragraph.3"))
+
+    @bulletList(List(
+        Html(messages("guidance.entry.section.1.list.1.bullet.1", link(
+            text = messages("guidance.entry.section.1.list.1.bullet.1.link"),
+            call = Call("GET", guidance.takingCommercialGoodsOnYourPerson)
+        ))),
+        Html(messages("guidance.entry.section.1.list.1.bullet.2", link(
+            text = messages("guidance.entry.section.1.list.1.bullet.2.link"),
+            call = Call("GET", guidance.exportingByPost)
+        )))
+    ))
+
+    @paragraphBody(messages("guidance.entry.section.1.paragraph.4", link(
+        text = messages("guidance.entry.section.1.paragraph.4.link.1"),
+        call = Call("GET", controllers.routes.GuidanceController.sendByRoro.url)
+    )))
+
+
+@* Section 2 *@
+
+    @heading(messages("guidance.entry.section.2.header.1"), "govuk-heading-l", "h2")
+    @heading(messages("guidance.entry.section.2.header.2"), "govuk-heading-m", "h3")
```

Comment:
My only doubt is if `sub-header`s should have as class `govuk-heading-m` or `govuk-heading-s`, because a part being in bold, in the text prototype they seem to have the same size of normal text.

We should agreed on this so to have uniformity there.

Reply:
I'm happy either way - I went with what was in the prototype:

        <h2 class="govuk-heading-l">Make the declaration yourself
          <h3 class="govuk-heading-m">Make a declaration on the new Customs Declaration Service (CDS) system</h3>
        </h2>

To be honest I don't like orphan subheaders - so if an improvement can be made here then great.

Reply:
I'll leave the decision to you :+1: 

I have used `govuk-heading-s` as class and `paragraphBody` as partial for the `complete_declaration` page.

If you decide for `heading(..., "govuk-heading-m")` please also amend it in `complete_declaration` (it's just one line) by using this ticket.

Reply:
Changed to -s for consistency.